veocean-data-farming/faq.md

```diff
@@ -0,0 +1,121 @@
+</details>
+
+<details>
+<summary>Can the reward function change during a given week?</summary>
+
+No. At the beginning of a new DF round (DF1, DF2, etc), rules are laid out, either implicitly if no change from previous round, or explicitly in a blog post if there are new rules. This is: reward function, bounds, etc. Then teams stake, buy data, consume, etc. And LPs are given DF rewards based on staking, DCV, etc at the end of the week. Overall cycle time is one week.
+
+Caveat: it’s no at least in theory! Sometimes there may be tweaks if there is community consensus, or a bug.
+</details>
+
+## veOCEAN FAQ
+<details>
+<summary>What is the official formula for the Linear Decay?</summary>
+
+The Linear Decay formula for veOCEAN can be expressed as follows in python.  
+```python
+FOUR_YEARS = 60 * 60 * 24 * 7 * 52
+
+veOcean_balance = OCEAN_amount_locked * (your_unlock_timestamp — current_unix_timestamp ) / FOUR_YEARS
+```
+
+To learn more about systems driving veOCEAN and Data Farming, please [visit our df-py github repository](https://github.com/oceanprotocol/df-py).
+</details>
+
+## Staking FAQs
+<details>
+<summary>What about passive stakers — people who just want to stake in one place and be done?</summary>
+
+Earnings are passive by default
+</details>
+
+<details>
+<summary>What about active stakers — people who want to do extra work and get rewarded?</summary>
+
+Ot works. Half the DF revenue goes to veOCEAN stake that users can allocate. Allocate well → more $$
+</details>
+
+## Third-Party Market FAQs
+<details>
+<summary>What do you think could be the monetization / incentive options for third party dApps and Data Markets that build on top of OCEAN?</summary>
+
+People pay fees on fixed-rate exchange and in consume. 3rd party markets can get a cut of both. More info at Ocean docs.
+</details>
+
+<details>
+<summary>Staking veOCEAN will be an important part in curating the most valuable datasets. Which earning opportunities will stakers have other than DF and fees generated from consume transactions?</summary>
+
+veOCEAN holders’ earning potential via fees and DF on their own could be quite substantial, eg given that a large portion of OCEAN supply is directed towards DF. No need for more ways per se. There won’t be a fee on locking OCEAN for veOCEAN, because adding that code would have meant changes to veCRV contracts, something we’re not doing for security reasons. Re ALGA or other apps making money on facilitating veOCEAN: there are several ways, the best way to get inspiration is to look at projects on top of base ve tokens, such as Convex, Aura, Redacted Cartel, and more.
+</details>
+
+## Pricing FAQs
+<details>
+<summary>In this scheme, can people stake on fixed-price datasets?</summary>
+
+Yes. They allocate their veOCEAN to datasets. Then DF rewards follow the usual DF formula: DCV * veOCEAN stake.
+</details>
+
```

Comment:
Add a new Q&A for something like "Can people stake on free datasets"?

Reply:
Done